tests/layer_tests/pytorch_tests/test_logaddexp.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestLogAddExp(PytorchLayerTest):
+    def _prepare_input(self, input1, input2, dtype="float32"):
+        """Prepare inputs for logaddexp testing"""
+        return (np.array(input1).astype(dtype), np.array(input2).astype(dtype))
+
+    def create_model(self, dtype=None):
+        import torch
+
+        dtype_map = {
+            "float32": torch.float32,
+            "float64": torch.float64,
+        }
+
+        class LogAddExpModel(torch.nn.Module):
+            def __init__(self, dtype=None):
+                super(LogAddExpModel, self).__init__()
+                self.dtype = dtype_map.get(dtype) if dtype else None
+
+            def forward(self, x, y):
+                if self.dtype:
+                    x = x.to(self.dtype)
+                    y = y.to(self.dtype)
```

Comment:
I think we don't need this casting. It is sufficient to generate input data of required type.

Reply:
please remove and the map as well